src/main/java/com/google/sps/servlets/YouTubeServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
```

Comment:
Missing Java unit tests. Please ask if you are not sure on what to actually test. Making unit tests a requirement before merging any code.